kpagacz/wiki-on-fire
# add(docker-compose): added a docker compose file for a database instance

* added a docker-compose file to run a mysql database instance


To test:
* install docker-compose (comes with Docker Desktop on Windows, on MacOS and Linux need to install separately)
* navigate to the new directory
* run
```bash
docker-compose --file development-env.yml up -d 
```
* the instance of mySQL database should run on port 3306.

I found it pretty useful when I was testing out the sequelize code.
